app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    parts = 3
+    result = split_integer(value, parts)
+    assert sum(result) == value, "sum(result) == value"
```

Comment:
by the cheklist its not correct
![git_check_1](https://github.com/mate-academy/py-split-integer/assets/154929153/4dca8779-41f4-43d7-8af2-e0b453f25561)